lib/scala/project-manager/src/main/resources/META-INF/native-image/org/enso/projectmanager/jni-config.json

```diff
@@ -1,15 +1,170 @@
 [
-  {
-    "name": "java.lang.ClassLoader",
-    "methods": [
-      { "name": "getPlatformClassLoader", "parameterTypes": [] },
-      { "name": "loadClass", "parameterTypes": ["java.lang.String"] }
-    ]
-  },
-  {
-    "name": "java.lang.ClassNotFoundException"
-  },
-  {
-    "name": "java.lang.NoSuchMethodError"
-  }
+{
+  "name":"com.sun.jna.Callback"
+},
+{
+  "name":"com.sun.jna.CallbackReference",
```

Comment:
This file shows how bloated code gets loaded in for as simple thing as finding the trashbin. Given the production code is always NI compiled, can't we use [SVM's C interface](https://github.com/JaroslavTulach/matrixultimate/blob/master/src/main/java/org/apidesign/demo/matrixultimate/svm/RawScientificLibrary.java#L14) instead?